components/operations-controller/internal/server/config.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 The Compass Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package server
+
+import (
+	"github.com/kyma-incubator/compass/components/system-broker/pkg/server"
+)
+
+// Settings type to be loaded from the environment
+type Config struct {
```

Comment:
Validate method is inherited from *server.Config and will get called as part of the Validation of config.Config. Thus MetricAddress and EnableLeaderElection fields are not validated though. If we want to validate those fileds a local Validate method should be introduced.

Reply:
`MetricAddress` and `EnableLeaderElection` are not mandatory properties and are pretty wide open in terms of possible values they could have assigned. If we have a need for explicit validation of them and have a clear understanding what an invalid value for each of them means, then yes, we could introduce a `Validate` method for this struct, have it call the embedded `*server.Config`'s Validate and then do explicit validation of the two properties.